ui/src/components/RORForm/index.tsx

```diff
@@ -367,7 +367,7 @@ const RORForm: React.FC<FormProps> = (props): React.ReactElement => {
                         {method === 'manual' && !isLinkValid ? (
                             <Components.Alert
                                 severity="ERROR"
-                                title="Please enter a valid URL."
+                                title='Reference link must be a valid URL starting with "http".'
```

Comment:
Think this shouldn't say 'Reference Link' (as this is uses for the funders/affiliates too) 

Please enter a valid URL starting with "http".

should be fine

Reply:
Oh God, missed that one :)). I updated it now